nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,125 @@
+import io
+import json
+import datetime
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = "Consolidates survey data from all nodes and exports the meta graph as a JSON file."
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--timestamp',
+            type=int,
+            action='store',
+            dest='timestamp',
+            help="All the data will be collected between this UNIX timestamp and two hours preceding it."
```

Comment:
Yes, only the latest one. But the timestamp ensures any data collected falls within the specified interval.